### DISCUSSION TITLE
Save password?

### DISCUSSION THREAD

**Discussion by Anonymous941** [Murmele/Gittyup#453] [Q&A]
How do I save my Gitea (Codeberg) password?  It prompts me to enter it every time.

**Comment by exactly-one-kas** [comment]
![grafik](https://user-images.githubusercontent.com/16035731/216254702-fb7b4f97-5282-42c0-9ce3-fddc3dac03b6.png)
Is the setting "Store credentials in secure storage" enabled?

**Reply by Anonymous941** [reply]
Yes, but it's still not saving.

**Reply by exactly-one-kas** [reply]
Which OS are you using? Passwords should be saved using GIT's credential helpers

**Reply by Anonymous941** [reply]
Edit: it stopped working again, see below

For some reason, it saved when I tried it today.

**Comment by dkebler** [comment]
I'm having this issue as well.  store credentials is checked but keeps asking for them.

Bug?

running arch linux manjaro

gittyup v1.2.2 - 2023-01-22

my origin is not github but my own gitea git server

**Reply by Anonymous941** [reply]
The issue came back up, and I'm using Codeberg (based on Gitea), so it's probably related to Gitea somehow

**Comment by dkebler** [comment]
Guess I need to investigate probably manually set up a credential store (as I did before all these gui clients) and then try pushing with the command line.  

I never had this issue with smartgit (saving credentials) with the same git server  and if I can sign in each time I don't see how the git server is the issue with saving a credential locally.  Unfortunately I don't have paid private repos on github so I can't use github to troubleshoot.  

Can any of the devs help point me to troubleshoot this issue?

**Reply by Anonymous941** [reply]
I'm using a Codeberg token, so probably you could reproduce the issue by testing it on Codeberg, either by self-hosting or making an account.

**Comment by dkebler** [comment]
What "secure" credential helper is used by default libsecret?   

on arch libsecret is in /usr/lib/git-core/git-credential-libsecret.  If gittyup uses it is that where it is expected?

**Comment by dkebler** [comment]
Ok I added keepassxc credential helper and it works  fine from the commandline.  How to get to work from gittyup?  

I'm going to start a new discussion item
